Tensile/Source/client/source/DataInitialization.cpp

```diff
@@ -379,11 +379,6 @@ namespace Tensile
                 m_maxBatch = std::max(m_maxBatch, numOfBatch);
             }
 
-            m_aMaxElements += m_aBufferOffset;
```

Comment:
Why this part is removed?
If we remove this, should we remove m_aBufferOffset as well?
It looks like m_aBufferOffset used only here.

Reply:
Offsets being added here created a bug where buffers would be allocated that were 2x the size of the offset. Offset is already handled in TensorDescriptor, and unnecessarily doubling the allocation made testing large offset problems impossible due to running out of memory. This change fixes the bug, but as you mention there may be additional clean-up work that could be done. I'll create a separate ticket for that.